modular_newhorizons/master_files/code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -0,0 +1,24 @@
+/singleton/reagent/toxin/potassium_chlorophoride
+	description = "A specific chemical based on Potassium Chloride to stop the heart for surgery. Not safe to eat!"
+
+/singleton/reagent/lexorin
+	description = "Lexorin temporarily stops respiration. Causes tissue damage."
+
+/singleton/reagent/soporific
+	description = "An effective hypnotic used to treat insomnia, can act as a sedative. Lasts three times longer when inhaled."
```

Comment:
"Soporific is one of the more common slow-acting sedatives utilized by medical institutions. As a highly diluted polysomnine, which results in slower and more gradual sedation, this makes the drug ideal at treating insomnia and anxiety disorders, however is generally not reliable for sedation in preparation for surgery except in high doses. Lasts three time as long when inhaled."